AzureAPIManagement/Utilities/Constants.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SmallStepsLabs.Azure.ApiManagement
+{
+    internal static class Constants
+    {
+        internal static class ApiManagement
+        {
+            internal static readonly string AccessToken = "SharedAccessSignature";
```

Comment:
Needs to be parameterized. We may publish this within different accounts / staging environments with automatic transforms